web/src/engine/main/src/keymanEngine.ts

```diff
@@ -104,8 +105,9 @@ export default class KeymanEngine<
     this.config = config;
     this.contextManager = contextManager;
 
-    this.interface = new KeyboardInterface(window, this, config.stubNamespacer);
-    this.core = new InputProcessor(config.hostDevice, worker, this.processorConfiguration());
+    const processorConfiguration = this.processorConfiguration();
+    this.interface = processorConfiguration.keyboardInterface as KeyboardInterface<ContextManager>;
```

Comment:
Cannot do in this form.  The form `app/browser` needs is not directly available to `engine/main` and we'd still have the bug being addressed by this PR.  Actually, it's the _other_ reason the bug even exists.

Granted, I could just set up a `protected` function to funnel it from `app/browser`'s subclassed variant... which is the role `processorConfiguration` is already meant to play.

In the subclass, we can't make assignments against `this` before the `super` call... at which point, this block runs.  Which is why it can't be set within the constructor for `app/browser`'s variant in time to do it this way.